### PR TITLE
Update stats component to use toLocaleString()

### PR DESCRIPTION
**Pull request by ehayman**
Thoughts on updating stats component to format numbers using toLocaleString() as opposed to leaving numbers entirely unformatted?

### CODE REVIEW

**Comment by rayrutjes** [comment]
Glad you bring it up.
I did so in the beginning and rolled back because of IE compatibility.
`toLocaleString ` is only in IE11+.

Not sure how much of an issue that is, and maybe we can do a check to test if the function exists.
WDYT?

**Comment by ehayman** [comment]
Ah, good point - I would argue though with < IE11 usage being at less than 1% of the total browser usage and steadily declining, we can safely ignore it: https://www.netmarketshare.com/browser-market-share.aspx?qprid=2&qpcustomd=0 End of support was announced by Microsoft 1.5+ years ago at this point.

That said, if you feel it's still important we can definitely just do a check - this is more of a high-level question about browser support than necessarily related to this one component. :)

**Comment by rayrutjes** [comment]
There you go @ehayman glad to hear we can move forward with this ;)

I have no idea why travis failed though. Will relaunch manually.

File: src/components/Stats.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default {
       return this.searchStore.query;
     },
     totalResults() {
-      return this.searchStore.totalResults;
+      return this.searchStore.totalResults.toLocaleString();
```
Comment (Haroenv) [inline src/components/Stats.vue]: I think we can have an argument to this function and forwarding the locale into the `toLocaleString()` call 🤔

**Comment by rayrutjes** [comment]
I just changed the commit message so that it can be picked up automatically by changelog while generating the new release.

Did you see @Haroenv 's comment @ehayman ? Do you want to add that or you think the current iteration is good as a starter?

**Comment by rayrutjes** [comment]
Lets do another iteration on @Haroenv suggestion in another PR if we want ;) or wait until the real need arises.

**Comment by ehayman** [comment]
Sorry just catching up on GH notifs :)

Thanks for merging / accepting!

**Pull request merged**